### PR TITLE
Add support for PHP 8

### PR DESCRIPTION
**Pull request by sebdesign**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ jobs:
 
       - name: Run linter
         run: composer run-script lint
+        if: matrix.php < '8.0'
```
Comment (mike182uk) [inline .github/workflows/ci.yml:35]: How come we don't want to run the linter on PHP8?

**Comment by mike182uk** [comment]
Thanks for this @sebdesign 

For some reason Scruitinizer is complaining butI can't see what the issue is?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (sebdesign) [reply .github/workflows/ci.yml:35]: php-cs-fixer does not run on PHP8 yet: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4702

**Comment by sebdesign** [comment]
The issue with Scrutinizer may was related to this: https://github.com/scrutinizer-ci/ocular/pull/52, but I'm not sure. Now that this PR was merged we can try to run it again.

**Comment by mike182uk** [comment]
Ah, all good @sebdesign Looks to be running fine on Scrutiniser now. Thanks for this 👍

**Pull request merged**